openedx/core/djangoapps/theming/models.py

```diff
@@ -0,0 +1,39 @@
+"""
+Django models supporting the Comprehensive Theming subsystem
+"""
+from django.db import models
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+
+class SiteTheme(models.Model):
+    """
+    This is where the information about the site's theme gets stored to the db.
+
+    `site` field is foreignkey to django Site model
+    `theme_dir_name` contains directory name having Site's theme
+    """
+    site = models.ForeignKey(Site, related_name='themes')
```

Comment:
So this has come up a couple times and a OneToOneField does make sense except there was a great idea from @mtyaka during our initial effort of introducing the ability to 'stack' themes in a sort of cascading approach.  We really like this idea and in order to build it we'd want the relationship to be a ForeignKey and not a OneToOneField.